src/harness/runner.ts

```diff
@@ -200,7 +196,7 @@ function handleTestConfig() {
 
         // TODO: project tests don"t work in the browser yet

```

Comment:
Is this still the case? Both the projects runner and browser IO impl got major overhauls.